app/Models/Location.php

```diff
@@ -208,6 +223,17 @@ public function parent()
             ->with('parent');
     }
 
+    /**
+    * Establishes the locations -> company relationship
+    *
+    * @author [T. Regnery] [<tobias.regnery@gmail.com>]
+    * @since [v6.0]
+    * @return \Illuminate\Database\Eloquent\Relations\Relation
+    */
+    public function company()
+    {
+        return $this->belongsTo('\App\Models\Company', 'company_id');
```

Comment:
Just to match the style of surrounding methods:
```suggestion
        return $this->belongsTo(\App\Models\Company::class, 'company_id');
```

Reply:
Fixed